conda/meta.yaml

```diff
@@ -26,7 +26,7 @@ test:
     - ts-dds
     - ts-idl {{ idl_version }}
     - ts-salobj {{ salobj_version }}
-    - tc-tcpip
+    - ts-tcpip
```

Comment:
don't you need a specific minimum version of ts-tcpip? you might want to provide some version information here at least a ">="